### PR TITLE
Adding Cocktail Sort

### PR DESCRIPTION
**Pull request by RaZorfalkon**
Added cocktail_sort.cs and updated the README file.

### CODE REVIEW

File: sorts/cocktail_sort.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        {
+            Console.WriteLine("Please enter some integers, separated by spaces:");
+            string input = Console.ReadLine();
+            string[] integers = input.Split(' ');
+            int[] data = new int[integers.Length];
+            for (int i = 0; i < data.Length; i++)
+                data[i] = int.Parse(integers[i]);
+            
+            Console.Write("\nUnsorted: ");
+            for (int i = 0; i < data.Length; i++)
```
Comment (christianbender) [inline sorts/cocktail_sort.cs]: You can write the print procedure shorter. [See here](https://stackoverflow.com/questions/16265247/printing-all-contents-of-array-in-c-sharp/16265268)

File: sorts/cocktail_sort.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            {
+                Console.Write(data[i] + " ");
+            }
+
+            CocktailSort(data);
+            
+            Console.Write("\nSorted: ");
```
Comment (christianbender) [inline sorts/cocktail_sort.cs]: Sames as above. Write it C#-like

**Comment by RaZorfalkon** [comment]
Thank's for your suggestion. I implemented a better way to print the array

**Comment by christianbender** [comment]
@RaZorfalkon Thanks for contributing

**Pull request merged**